global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What are the issues.  The repo should only use the cli found in tools\dotnet/cli.  If we are not including the path to that when we need to then there is a bug.

So please remove this file. the rest looks great.